docs/metrics.md

```diff
@@ -192,23 +192,31 @@ soroban.host-fn-op.failure                   | meter     | number of failed `Inv
 soroban.host-fn-op.exec                      | timer     | total time spent during the `InvokeHostFunctionOp`
 soroban.restore-fprint-op.read-ledger-byte   | meter     | number of `LedgerEntry` bytes accessed (read or modified) during the `RestoreFootprintOp`
 soroban.restore-fprint-op.write-ledger-byte  | meter     | number of `LedgerEntry` bytes modified during the `RestoreFootprintOp`
+soroban.restore-fprint-op.exec               | timer     | total time spent during the `RestoreFootprintOp`
 soroban.ext-fprint-ttl-op.read-ledger-byte   | meter     | number of `LedgerEntry` bytes accessed (read or modified) during the `ExtendFootprintTTLOp`
+soroban.ext-fprint-ttl-op.exec               | timer     | total time spent during the `ExtendFootprintTTLOp`
+soroban.ledger.tx-count                      | histogram | number of soroban transactions per ledger
 soroban.ledger.cpu-insn                      | histogram | metered cpu instructions per ledger
```

Comment:
The per-ledger metric comments need to be update now to reflect that they're based on declared values.

Reply:
Done.